### ISSUE TITLE
Doesn't work at all inside vue component templates

### ISSUE THREAD

**Issue by saikojosh** [mradionov/eslint-plugin-disable#10]
**App.vue:**

```javacript
<template>

	<div id="app" class="example"><span>Some info: </span>{{ msg }}</div>

</template>

<script>
	/* eslint-plugin-disable node */  // <-- this does not work.

	export default {
		data: function () {
			return {
				msg: `Hello, World!`,
			};
		},
	};

</script>

<style lang="scss">

	.example {
		color: green;
		border: 1;

		span {
			text-decoration: line-through;
		}
	}

</style>
```

**Expected Behaviour:** The "eslint-plugin-node" plugin is disabled when the inline comment `/* eslint-plugin-disable node */` is included in the Vue template.

**Actual Behaviour:** The inline comment is ignored and the "eslint-plugin-node" plugin is not disabled for the Vue template.

**Comment by saikojosh** [comment]
Used in conjunction with "eslint-plugin-vue".

**Comment by mradionov** [comment]
By default only files with `.js` and `.ts` extensions are handled. Make sure to add `.vue` to the list of extensions when configuring the plugin. [See docs](https://github.com/mradionov/eslint-plugin-disable#processed-extensions)

```
{
  "settings": {
    "eslint-plugin-disable": {
      "extensions": [".js", ".ts", ".vue"]
    }
  }
}
```

**Comment by saikojosh** [comment]
Sadly that makes no difference, it still doesn't work in .vue files.

**Comment by mradionov** [comment]
Thanks for the info, let me figure it out.

**Comment by mradionov** [comment]
I've used the same setup from your another issue (https://github.com/mradionov/eslint-plugin-disable/issues/9), and got plugin to work. 

The only thing I was struggling with is to make eslint see `.vue` files. I had to add command line option when running eslint to parse `.vue` extension:

```
eslint --ext js,vue foo.vue
```

Here is the output without disabling:

![image](https://user-images.githubusercontent.com/2007370/32797911-45d1f836-c984-11e7-85d5-1d24a9732309.png)

And if I add `/* eslint-disable-plugin node */` after `<script>`:

![image](https://user-images.githubusercontent.com/2007370/32797937-5ffc11c4-c984-11e7-80d1-41a9c35f24bd.png)

All rules with `node/` prefix were removed from the output.

I am not sure why I am unable to reproduce both issues you have.
